packages/test/test-utils/src/opProcessingController.ts

```diff
@@ -136,7 +136,7 @@ class DeltaManagerMonitor extends DeltaManagerToggle {
 
         deltaManager.on("connect", (details) => this.connect(details.clientId));
         deltaManager.on("disconnect", (reason) => {
-            assert(this.clientId !== undefined);
+            // assert(this.clientId !== undefined);
```

Comment:
same with this assert, let's make sure this only happens if storageOnly === ture